From 7a00741653f66c0b46dd6c2a41f6d7a209791105 Mon Sep 17 00:00:00 2001
From: Jiazhi Huang <huangjia2hi@163.com>
Date: Tue, 11 Aug 2026 17:37:43 +0800
Subject: [PATCH 13228/15333] Documentation: hwmon: (nct6775) Add missing
 NCT6797D and NCT6798D

The NCT6797D and NCT6798D chips are already supported by the driver
but were missing from the documentation.

Signed-off-by: Jiazhi Huang <huangjia2hi@163.com>
Link: https://lore.kernel.org/r/20260811093744.12199-2-huangjia2hi@163.com
Signed-off-by: Guenter Roeck <linux@roeck-us.net>
---
 Documentation/hwmon/nct6775.rst | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/Documentation/hwmon/nct6775.rst b/Documentation/hwmon/nct6775.rst
index 9d7a10de61a7..6725f8ff98e0 100644
--- a/Documentation/hwmon/nct6775.rst
+++ b/Documentation/hwmon/nct6775.rst
@@ -80,6 +80,22 @@ Supported chips:
 
     Datasheet: Available from Nuvoton upon request
 
+  * Nuvoton NCT6797D
+
+    Prefix: 'nct6797'
+
+    Addresses scanned: ISA address retrieved from Super I/O registers
+
+    Datasheet: Available from Nuvoton upon request
+
+  * Nuvoton NCT6798D
+
+    Prefix: 'nct6798'
+
+    Addresses scanned: ISA address retrieved from Super I/O registers
+
+    Datasheet: Available from Nuvoton upon request
+
   * Nuvoton NCT6796D-S/NCT6799D-R
 
     Prefix: 'nct6799'
-- 
2.55.0

